types/tarantool-driver/index.d.ts

```diff
@@ -0,0 +1,58 @@
+// Type definitions for tarantool-driver 3.0
+// Project: https://github.com/tarantool/node-tarantool-driver
+// Definitions by: Evgeni Zharkov <https://github.com/zharkov-eu>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { EventEmitter } from "events";
+
+export interface TarantoolOptions {
+  host?: string;
+  port?: number;
+  username?: string;
+  password?: string;
+  reserveHosts?: string[];
+  beforeReserve?: number;
+  timeout?: number;
+  retryStrategy?: (times: number) => number;
+  lazyConnect?: boolean;
+}
+
+export default class TarantoolConnection extends EventEmitter {
```

Comment:
the source library uses `module.exports =` not `export default`. The equivalent typescript construct is `export = TarantoolConnection`, which is then imported with `import Tarantool = require('tarantool-driver')`

Reply:
Default export changed in f89faea